pkgs/development/tools/verible/default.nix

```
@@ -0,0 +1,85 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, buildBazelPackage
+, bazel_4
+, flex
+, bison
+, python3
+}:
+
+buildBazelPackage rec {
+  pname = "verible";
+  version = "v0.0-2172-g238b6df6";
+
+  # These environment variables are read in bazel/build-version.py to create
+  # a build string. Otherwise it would attempt to extract it from .git/.
+  GIT_DATE = "2022-08-08";
+  GIT_VERSION = version;
+
+  src = fetchFromGitHub {
+    owner = "chipsalliance";
+    repo = "verible";
+    rev = version;
```

```suggestion
  version = "0.0-2172-g238b6df6";

  # These environment variables are read in bazel/build-version.py to create
  # a build string. Otherwise it would attempt to extract it from .git/.
  GIT_DATE = "2022-08-08";
  GIT_VERSION = "v${version}";

  src = fetchFromGitHub {
    owner = "chipsalliance";
    repo = "verible";
    rev = "v${version}";
```
usually we strip the v prefix which I forgot earlier.

---

Done.